package.json

```diff
@@ -115,7 +112,7 @@
         ],
         "clearMocks": true,
         "transformIgnorePatterns": [
-            "node_modules/(?!(@ckeditor|ckeditor5|array-move|lodash-es|@react-leaflet|react-leaflet)/)"
+            "node_modules/(?!(@ckeditor|ckeditor5|array-move|lodash-es)/)"
```

Comment:
`array-move` and `lodash-es` still required to be transformed for running `jest` tests. Link from maintainer of array-move about esm packages: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c